### PR TITLE
feat: NOD-18 pagopa mig pg

### PR DESCRIPTION
**Pull request by cesarecaccuri**
### List of changes

moved **nodo** liquibase changelogs to src/psql/nodo and organized by version 
added **nodo-db-migration-pipelines.yml** to create a pipeline which runs liquibase on the target postgresql db
added **nodo-db-schema-pipelines.yml** to inizialize **nodo** schemas/users on the target postgresql db

### Motivation and context

Needed to initialize/migrate a postgresql db for **nodo**

### Type of changes

- [x] Add new resources
- [ ] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [x] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [x] No

### Other information

---

### CODE REVIEW

File: .devops/nodo-db-migration-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+  ONLINE_USERNAME: online
+  ONLINE_SCHEMA: online
+  RE_USERNAME: re
+  RE_SCHEMA: re
+  WFESP_USERNAME: wfesp
+  WFESP_SCHEMA: wfesp
+
+stages:
+  - stage: prd_approval
+    pool: server
```
Comment (pasqualespica) [inline .devops/nodo-db-migration-pipelines.yml:61]: why ? `pool: server`

File: .devops/nodo-db-migration-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+        pool: server
+        condition: eq('${{ parameters.k8sEnv }}', 'prd')
+        displayName: "Approval"
+        timeoutInMinutes: 6
+        steps:
+          - task: ManualValidation@0
+            timeoutInMinutes: 5
+            inputs:
+              notifyUsers: |
+                test@test.com
```
Comment (pasqualespica) [inline .devops/nodo-db-migration-pipelines.yml:75]: change with slack and google internal grp

File: .devops/nodo-db-migration-pipelines.yml [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+    default: dev
+    values:
+      - dev
+      - uat
+      - prd
+  - name: dbVersion
+    displayName: Database Version
+    type: string
+  - name: lbContexts
+    displayName: Liquibase Contexts
```
Comment (pasqualespica) [inline .devops/nodo-db-migration-pipelines.yml:22]: add comment to describe it

File: .devops/nodo-db-migration-pipelines.yml [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+  - name: k8sEnv
+    displayName: K8s Environment
+    type: string
+    default: dev
+    values:
+      - dev
+      - uat
+      - prd
+  - name: dbVersion
+    displayName: Database Version
```
Comment (pasqualespica) [inline .devops/nodo-db-migration-pipelines.yml:19]: add comment to describe it ( ex. refers to versions current repo under `liquibase/changelog/` folder

**Review by pasqualespica — CHANGES_REQUESTED** [review]
before the manual approval activity might it be useful to add a liquid base activity to see the changes it would make?

File: .devops/nodo-db-migration-pipelines.yml (hunk 1 above)
Comment (cesarecaccuri) [reply .devops/nodo-db-migration-pipelines.yml]: mistake,probably not needed,removed

File: .devops/nodo-db-migration-pipelines.yml (hunk 2 above)
Comment (cesarecaccuri) [reply .devops/nodo-db-migration-pipelines.yml]: done

File: .devops/nodo-db-migration-pipelines.yml (hunk 3 above)
Comment (cesarecaccuri) [reply .devops/nodo-db-migration-pipelines.yml:24]: done

File: .devops/nodo-db-migration-pipelines.yml (hunk 4 above)
Comment (cesarecaccuri) [reply .devops/nodo-db-migration-pipelines.yml:20]: done

**Review by pasqualespica — APPROVED** [review]
great 👍

**Pull request merged**